### DISCUSSION TITLE
disable logging

### DISCUSSION THREAD

**Discussion by jamesbradlee** [paketo-buildpacks/feedback#250] [Java Team]
Hey, is there a way to disable all the logs from Paketo? This does not fit in to our log format, and we're not intersted in seeing this information.
![image](https://github.com/paketo-buildpacks/feedback/assets/125522276/617fb01b-b8dd-4099-8a54-d12ce6189b67)
![image](https://github.com/paketo-buildpacks/feedback/assets/125522276/5c945891-3191-47d3-9f78-6726aa0c4f51)

**Comment by dmikusa** [comment]
That output comes from the JVM. It is the Java NMT that we enable by default. You can disable this by setting the environment variable `BPL_JAVA_NMT_ENABLED=false` when you run your application.

**Comment by jamesbradlee** [comment]
I'll give that a try! Thank you :)

**Comment by jamesbradlee** [comment]
That did indeed work! I'm still seeing some other logs, do you happen to know how to remove those as well?
![image](https://github.com/paketo-buildpacks/feedback/assets/125522276/748d2fda-4410-4fd6-af8e-fe4e86355383)

**Reply by dmikusa** [reply]
No. Sorry. Some of that comes from the buildpack itself and logging is not optional. Some comes from the JVM, also not optional.

It looks like some might come from OTel as well and that I don't know.

**Reply by jamesbradlee** [reply]
Yes, the last one is from OTel :)

**Reply by jamesbradlee** [reply]
I understand! Thanks for the feedback!
